lib/Master.php

```diff
@@ -143,10 +143,14 @@ public function handleLoginRequest(
 
 		$this->logger->debug('handleLoginRequest: uid is: ' . $uid);
 
-		// let the admin of the master node login, everyone else will redirected to a client
-		$masterAdmins = $this->config->getSystemValue('gss.master.admin', []);
-		if (is_array($masterAdmins) && in_array($uid, $masterAdmins, true)) {
-			$this->logger->debug('handleLoginRequest: this user is a local admin so ignore');
+		// let local account login, everyone else will redirected to a client
```

Comment:
```suggestion
// let local account and admin of the master node login, everyone else will redirected to a client
```

Reply:
admin are accounts :)

Reply:
it is more that the feature is currently limited to admin like the configuration parameter might let us think it is